//注册用户请求
message RegUserReq
{
    //用户名
    required string name = 1;
    //密码
    required string password = 2;
}
